### PR TITLE
Review of Packit as a Service article

### PR DESCRIPTION
**Pull request by eliskasl**

### CODE REVIEW

**Comment by eliskasl** [comment]
@phracek please take a look

File: content/packit-as-a-service.md [hunk 1]
```diff
@@ -4,31 +4,33 @@ date: 2019-06-22
...
+Using Packit Service is very straightforward: add one [config
+file](https://github.com/packit-service/packit/blob/master/docs/configuration.md)
+to your repository together with the RPM spec file and you're good to go. 
 
 Packit Service validates your pull requests by building your software in Fedora OS.
-Once the builds are done, packit lets you know how to install the
+Once the builds are done, Packit lets you know how to install the
 RPMs inside your environment so you can try the changes before merging them.
 
+Packit is also available as a [CLI tool](https://github.com/packit-service/packit/blob/master/README.md),
```
Comment (phracek) [inline content/packit-as-a-service.md]: I would prefer to use a tool. We have had a discussion with @jpopelka about it (https://github.com/packit-service/packit.dev/pull/15#discussion_r297253118)

File: content/packit-as-a-service.md [hunk 2]
```diff
@@ -4,31 +4,33 @@ date: 2019-06-22
...
 There are two ways to install Packit-as-a-Service GitHub app into your repository.
 
 ### Install from Packit-as-a-Service page
-1. From the [Packit-service page](https://github.com/apps/packit-as-a-service) page,
-in the left sidebar, click "Install App"
+1. From the [Packit-service page](https://github.com/apps/packit-as-a-service),
+in the left sidebar, click "Install App" (alternatively, click "Configure" in the right sidebar)
 2. Click "Install" next to the organization or user account containing the correct repository
-3. Install the "Packit-as-a-Service" on all repositories or to certain repositories
+3. Install "Packit-as-a-Service" to repositories where you wish to use the service
```
Comment (phracek) [inline content/packit-as-a-service.md]: I would use the original section. `on all repositories or to certain repository`.

File: content/packit-as-a-service.md (hunk 1 above)
Comment (TomasTomecek) [reply content/packit-as-a-service.md]: @jpopelka why not mention the CLI tool here?

Comment (eliskasl) [reply content/packit-as-a-service.md]: It read "tool" before which is pretty much the same as CLI tool. But let me know and I remove it completely.

File: content/packit-as-a-service.md (hunk 2 above)
Comment (eliskasl) [reply content/packit-as-a-service.md]: It sounded weird - it's obvious that you use it either on one, two or as many repos as you want.  Or is it not an expected option?

**Comment by phracek** [comment]
@eliskasl Feel free to merge it.
Before merging exec `make generate` which removes docs/ directory and generates a new one. `docs` directory has to be included in the merge request.

File: content/packit-as-a-service.md (hunk 1 above)
Comment (TomasTomecek) [reply content/packit-as-a-service.md]: I'm personally +1 on using the wording "CLI tool".

**Comment by eliskasl** [comment]
^ the force push above should contain regenerated docs btw

**Comment by TomasTomecek** [comment]
I pushed your changes from the CLI since you removed a ton of stuff we probably want to keep.

Thank you!

**Pull request closed without merging**